portfolio/src/main/webapp/script.js

```diff
@@ -19,28 +19,55 @@
  * Fetches comments from the server and adds it to the DOM
  */
 function getComments() {
-    fetch("/data").then(response => response.json()).then((commentsData) => {
+    fetch("/comments").then(response => response.json()).then((commentsData) => {
         const commentsContainer = document.getElementById("comments-container");
         commentsContainer.innerHTML = '';
-        for (var i = 0; i < commentsData.length; i++) {
-            var commentData = commentsData[i];
+        for (const i = 0; i < commentsData.length; i++) {
+            const commentData = commentsData[i];
 
-            // Create new comment element
-            var boldElement = document.createElement("b");
-            var boldNode = document.createTextNode("Comment " + (i + 1) + " - ");
-            boldElement.appendChild(boldNode);
+            // Create new comment elements
+            const commentHeader = createCommentHeader(commentData);
+            const commentContent = createCommentContent(commentData);
 
-            var commentElement = document.createElement("h4");
-            var textNode = document.createTextNode(commentData.comment);
-            commentElement.appendChild(boldElement);
-            commentElement.appendChild(textNode);
-
-            // Add new comment element to comments section
-            commentsContainer.appendChild(commentElement);
+            // Add new comment to comments section
+            commentsContainer.appendChild(commentHeader);
+            commentsContainer.appendChild(commentContent);
         }
     });
+
+}
+
+/* Creates header for comment containing name in bold and timestamp in italics */
+function createCommentHeader(commentData) {
+    const headerElement = document.createElement("h4");
+    
+    const nameElement = document.createElement("b");
+    nameElement.appendChild(document.createTextNode(commentData.name));
+
+    const sep = document.createTextNode(" ~ ");
+
+    const tsElement = document.createElement("i");
+    tsElement.appendChild(document.createTextNode(commentData.timeStamp));
+
+    headerElement.appendChild(nameElement);
+    headerElement.appendChild(sep);
```

Comment:
Could we inline this for better readability? Such as 
headerElement.appendChild(document.createTextNode(" ~ " ));

Reply:
Fixed!